src/main/archetype/ui.apps/src/main/content/jcr_root/apps/__appId__/components/page/customfooterlibs.html

```diff
@@ -20,6 +20,7 @@
 #end
 #if ( $includeFormsheadless == "y" )
     <sly data-sly-call="${symbol_dollar}{clientlib.js @ categories='${appId}.forms.react', async=true}"/>
+    <sly data-sly-test="${symbol_dollar}{!wcmmode.edit}" data-sly-call="${symbol_dollar}{clientlib.js @ categories='core.forms.components.runtime.all', async=true}"/>
```

Comment:
clientlibs should be included outside of headless flag. if includeForms or includeFormsEnrollment or communication is true.